package/gluon-respondd/src/respondd.c

```diff
@@ -208,6 +208,82 @@ static struct json_object * get_memory(void) {
 	return ret;
 }
 
+static struct json_object * get_stat(void) {
+	FILE *f = fopen("/proc/stat", "r");
+	if (!f)
+		return NULL;
+
+	struct json_object *stat = json_object_new_object();
+	struct json_object *ret = NULL;
+
+	char *line = NULL;
+	size_t len = 0;
+
+	while (getline(&line, &len, f) >= 0) {
+		char label[32];
+
+		if (sscanf(line, "%31s", label) != 1){
+			goto invalid_stat_format;
+		}
+
+		if (!strcmp(label, "cpu")) {
+			unsigned long long user, nice, system, idle, iowait, irq, softirq;
+			if(sscanf(line, "%*s %llu %llu %llu %llu %llu %llu %llu",
```

Comment:
Please add a space after the *if* to keep the style consistent (also in the following *sscanf* lines).